packages/components/src/ActionList/ActionList.scss

```diff
@@ -19,7 +19,7 @@ $tc-action-list-item-border-size: 2px !default;
 			.btn.btn-link {
 				padding: $padding-normal;
 				text-overflow: inherit;
-				text-transform: uppercase;
+				text-transform: inherit;
```

Comment:
i had a discussion with Stéphanie B. about this change and i'm not sure we need to do it now. (need before to fix existing translation in our projects). please see with her before merge your PR

Reply:
As discussed with Stephanie, everything is OK. Screenshots will be made on the INT stack.